src/Sylius/Bundle/AdminBundle/Resources/views/Order/Show/Content/_sections.html.twig

```diff
@@ -1,20 +1,11 @@
-{% set main_hook = hook_name(hookable_data.parent_main_hook, 'sections') %}
-{% set fallback_hook = hook_name(hookable_data.parent_fallback_hook, 'sections') %}
-
 <div class="page-body">
     <div class="container-xl">
         <div class="row">
             <div class="col-12 col-lg-3 order-lg-1">
-                {% hook [main_hook ~ '#right', fallback_hook ~ '#right'] with hookable_data|merge({
-                    parent_main_hook: main_hook ~ '#right',
-                    parent_fallback_hook: fallback_hook ~ '#right',
-                }) %}
+                {% hook 'sections#right' %}
```

Comment:
Shouldn't the left be first?

Reply:
🤷🏼‍♂️

We might check that, but it's out of the scope here. I'll check that later, and create a task if needed.